pypsa/components.py

```diff
@@ -480,7 +546,7 @@ def df(self, component_name):
         """
         return getattr(self, self.components[component_name]["list_name"])
 
-    def pnl(self, component_name):
+    def pnl(self, component_name: str) -> Dict:
```

Comment:
```suggestion
    def pnl(self, component_name: str) -> dict:
```

Reply:
@fneum 
Just FYI, there is `dict` and `from typing import Dict` for type checks. Both do the same but we wanna use the built-in `dict`.
But this refers to our PyPSA special `Dict` object (should be renamed):

https://github.com/PyPSA/PyPSA/blob/8093c685c87d9666e7e7dd3bdb35f7a4ca29f12f/pypsa/descriptors.py#L22-L69